CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 0.7.8
 -----
 
+* Data directory should not be created on import (Issue #2018)
```

Comment:
`"The SunPy data directory "~/sunpy" is no longer created until it is used (issue #2018)"` would be better. Changelogs should describe the fix rather than the bug.